uavcan/equipment/air_data/1027.RawAirData.uavcan

```diff
@@ -0,0 +1,29 @@
+#
+# Raw Air Data.
+#
+
+#
+# Heater State
+# 
+uint8 HEATER_UNAVAILABLE    = 0
+uint8 HEATER_WORKING        = 1
+uint8 HEATER_OVERCURRENT    = 2
+uint8 HEATER_OPENCIRCUIT    = 3
+uint8 heater_state
+
+#
+# Pressure Data
+#
+float32 static_pressure                 # Pascal
+float32 differential_pressure           # Pascal
+
+#
+# Temperature Data
+#
+float16 sensor_temperature              # Kelvin
+float16 static_air_temperature          # Kelvin
```

Comment:
Should we clarify here that we're talking about a static air temperature sensor rather than just a plain temperature sensor, since OAT sensors may incorporate corrections for the distortions caused by the flow of air?

> This field contains the raw temperature reading from the externally mounted static air temperature sensor, or, in the absence of one, the raw temperature of the pressure sensor.